lms/djangoapps/certificates/signals.py

```diff
@@ -58,7 +58,7 @@ def _listen_for_passing_grade(sender, user, course_id, **kwargs):  # pylint: dis
 
 
 @receiver(LEARNER_NOW_VERIFIED, dispatch_uid="learner_track_changed")
-def _listen_for_track_change(sender, user, **kwargs):  # pylint: disable=unused-argument
+def _listen_for_track_change(sender, user, verification_status, **kwargs):  # pylint: disable=unused-argument
```

Comment:
I would actually make the query for the verification inside of this function.  There's a convenient class method which tells you the status directly:
```python
expected_verification_status, _ = SoftwareSecurePhotoVerification.user_status(user)
```